src/components/SignName.vue

```diff
@@ -182,7 +201,12 @@ export default {
             .catch(function (error) {
                 console.log(error)
             })
-        }
+        },
+        checkFormFilled() {
+      // check if the form is fully filled here
+      this.isFormFilled = !!this.checkForm()
+      bus.$emit('formFilled', this.isFormFilled);
```

Comment:
I don't think you need to import bus. You should have access to emit with `this.$emit`